.github/workflows/dotnetcore.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - name: Setup .NET Core
         uses: actions/setup-dotnet@v1
         with:
-          dotnet-version: 5.0.102
+          dotnet-version: 6.0.x
```

Comment:
The `.x` means "use the latest released version". This seems useful, so we automatically apply e.g. security patches => Perlang will always be built (and include it its snapshots) on the latest, stable 6.0.x patch release.